store/tikv/mock-tikv/mvcc.go

```diff
@@ -547,6 +598,35 @@ func (s *MvccStore) RawScan(startKey, endKey []byte, limit int) []Pair {
 	return pairs
 }
 
+// MvccGetByStartTS gets mvcc info for the primary key with startTS
+func (s MvccStore) MvccGetByStartTS(starTS uint64) *kvrpcpb.MvccInfo {
```

Comment:
I think you need to add _startKey_ and _endKey_ arguments to restrict the method in a single region.